src/user.cpp

```diff
@@ -709,6 +717,7 @@ int User::needversioning(attr_t at)
         case ATTR_GEOLOCATION:
         case ATTR_MY_CHAT_FILES_FOLDER:
         case ATTR_PUSH_SETTINGS:
+        case ATTR_ALIAS:
```

Comment:
Since this attribute should be updated always based on the latest version, we should consider versioning. Otherwise, we can overwrite aliases set by a different client if we haven't received yet the corresponding actionpacket that invalidates our version and triggers the corresponding fetch.
Remember the `!` modifier in the name means the API doesn't store the history of values, but we can avoid race conditions thanks to sending to API our version, and API will reject the update if we are not up to date.
I'll change it to versioning.